homeassistant/components/modem_callerid/sensor.py

```diff
@@ -42,6 +46,11 @@ async def _async_on_hass_stop(event: Event) -> None:
     platform = entity_platform.async_get_current_platform()
 
     platform.async_register_entity_service(SERVICE_REJECT_CALL, {}, "async_reject_call")
+    _LOGGER.warning(
```

Comment:
Logging the warning here will log the warning even if the user doesn't use the deprecated service. I'd recommend moving the log to the service handler.

Reply:
#67223